onnxruntime/core/providers/cpu/object_detection/roialign.cc

```diff
@@ -270,7 +269,7 @@ void RoiAlignForward(
       }    // for ph
     }      // for c
   };       // for n
-  if (ttp != nullptr) const_cast<ThreadPool*>(ttp)->ParallelFor(static_cast<int32_t>(n_rois), work_object);
+  ThreadPool::ParallelFor(ttp, static_cast<int32_t>(n_rois), work_object);  
 }
```

Comment:
So if ttp was nullptr this method just ignored that and did nothing?

Reply:
Was a bug.